app/components/navigation_header/header.tsx

```diff
@@ -168,7 +166,7 @@ const Header = ({
     const containerAnimatedStyle = useAnimatedStyle(() => ({
         height: defaultHeight,
         paddingTop: insets.top,
-    }), [defaultHeight, lockValue]);
+    }), [defaultHeight]);
```

Comment:
remove dependency for lockValue, where it's not being used.

Reply:
so what was the reason this was used here? do you know?

Reply:
Not sure, traced back to a few commits behind, and it's always been there.  And in reality, defaultHeight and insets.top are not animated shared value, so it's not worth making this animated style.